### ISSUE TITLE
TrackMe bug report - splk-dhm/splk-mhm entity view host Metadata filter do not apply when hybrid tracker was created manually in a tenant (opposed to created during the Virtual Tenant creation phase)

### ISSUE THREAD

**Issue by sebwurl** [trackme-limited/trackme-report-issues#19]
In version 2.0.7 the DHM entity view does not use the host/entity as a filter in searches.
Here is an example for the first single value:
![image](https://user-images.githubusercontent.com/85566586/220678889-d5795512-299a-4e24-a588-e1aed1885e38.png)

![image](https://user-images.githubusercontent.com/85566586/220679268-71a3f48a-474c-489e-92b0-ab5d05dbd5a0.png)

This is also true for the other KPIs and the graps below.

**Comment by guilhemmarchand** [comment]
Thank you @sebwurl for reporting, will take of this ASAP!

**Comment by guilhemmarchand** [comment]
@sebwurl 

Could you the following REST call and share the result in your context?

```
| trackme url="/services/trackme/v2/splk_dhm/dh_entity_info" mode="post" body="{'tenant_id': 'test002', 'object': 'key:host|eventgen-generator-1'}"
```

Example of result:

```
{
	"account": "local",
	"search_mode": "tstats",
	"breakby_key": "host",
	"breakby_value": "eventgen-generator-1",
	"index": "siem_radius2,siem_radius3",
	"sourcetype": "eventgen:radius2,eventgen:radius3",
	"search_constraint": "index IN (siem_radius2,siem_radius3) sourcetype IN (eventgen:radius2,eventgen:radius3) host=\"eventgen-generator-1\"",
	"splk_dhm_overview_timechart": "| tstats count, max(_indextime) as indextime, max(_time) as maxtime where index IN (siem_radius2,siem_radius3) sourcetype IN (eventgen:radius2,eventgen:radius3) host=\"eventgen-generator-1\" by _time, index, sourcetype span=1s | eval delta=(indextime-_time), event_lag=(now() - maxtime) | timechart span=15m sum(count) as events_count, avg(delta) as avg_lag_sec",
	"splk_dhm_overview_pie_root_search": "| tstats count where index IN (siem_radius2,siem_radius3) sourcetype IN (eventgen:radius2,eventgen:radius3) host=\"eventgen-generator-1\" by index, sourcetype",
	"splk_dhm_overview_root_search": "| tstats count, max(_indextime) as indextime, max(_time) as maxtime where index IN (siem_radius2,siem_radius3) sourcetype IN (eventgen:radius2,eventgen:radius3) host=\"eventgen-generator-1\" by _time, index, sourcetype span=1s | eval delta=(indextime-_time), event_lag=(now() - maxtime) | stats perc95(delta) as perc95_lag, avg(delta) as avg_lag, latest(event_lag) as event_lag",
	"splk_dhm_raw_search": "search?q=index%20IN%20%28siem_radius2%2Csiem_radius3%29%20sourcetype%20IN%20%28eventgen%3Aradius2%2Ceventgen%3Aradius3%29%20host%3D%22eventgen-generator-1%22"
}
```

FYI, this endpoints define the root search constraint used in the UI.

**Comment by guilhemmarchand** [comment]
Note:

- This happens when the Hybrid Tracker was created post Virtual Tenant creation (manually)
- This affects both splk-dhm / splk-mhm
- entities should be created as a standard with: 
```
key:host|<value>
```
Instead these are created in the raw format which is symptomatic of the issue.

**Comment by sebwurl** [comment]
This how it looks when the tracker from the initial tenant creation is used:

![image](https://user-images.githubusercontent.com/85566586/220850381-facd9fa5-85cf-4139-9a75-8bc30200afea.png)

![image](https://user-images.githubusercontent.com/85566586/220851063-0d76458a-211c-441a-8332-ac37a12b54f4.png)

After re-creating the the tracker manually it looks like this:
![image](https://user-images.githubusercontent.com/85566586/220853571-dcb7784a-63ed-42c8-8f52-6b59b3f421e9.png)


![image](https://user-images.githubusercontent.com/85566586/220853394-b55a78d0-9b0e-4953-96da-3b0c82d86893.png)


I was wondering that the object name changed, but though this new raw format was intended.

**Comment by guilhemmarchand** [comment]
The first was intentended @sebwurl 

```
key:host|<host>
```

If using a different metadata, say region, it would be:

```
key:region|<region value>
```

Validating the fix as we talk

**Comment by guilhemmarchand** [comment]
This happens only when manually creating the hybrid tracker (not during the tenant creation phase)
But I have isolated the issue already.

**Comment by guilhemmarchand** [comment]
Fixed in 2.0.8 @sebwurl 

You need to either:

- delete and create the tenant
- delete the hybrid tracker, delete all entities, create a new hybrid tracker and get entities to be discovered

Thank you a thousand times for your incredible bug findings capabilities, cannot wait to see more from you ;-)
Appreciated your help
